src/QMCHamiltonians/HamiltonianFactory.cpp

```diff
@@ -176,8 +177,18 @@ bool HamiltonianFactory::build(xmlNodePtr cur)
         targetH->addOperator(std::move(hs), "Grid", true);
       }
     }
+    else if (cname == "flux")
+    {
+      if (potName.size() < 1)
```

Comment:
```suggestion
      if (potName.empty())
```

Reply:
I'm thinking of adding `legacy_estimator`? there is no guarantee we make the legacy ones working but at least users may have a chance to force it.

Reply:
> I'm not in favor of that as it will inhibit refactoring QMCHamiltonian. If there is a batched version the legacy version should be disabled for a batched run.

If there is a batched version, the old input through Hamiltonian should be blocked. What if not? I would prefer giving user a chance to try out things without modifying the source code.